app/controllers/api/v1/project_roles_controller.rb

```diff
@@ -10,9 +10,13 @@ def resource_name
   end
 
   def update
-    super do
-      UserAddedToProjectMailerWorker.perform_async(api_user.id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
-    end
+    UserAddedToProjectMailerWorker.perform_async(acl_user_id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
```

Comment:
First, no. Should come after, but the one in #create doesn't get called in the spec with `super do`. I tried `super do |acl|` as well, since other examples of this behavior in Panoptes passed a block parameter as well, but that #create one still never gets called. Still looking into this.